tools/utils/seed/template_locals.ts

```diff
@@ -1,21 +1,46 @@
+import * as util from 'gulp-util';
 import { argv } from 'yargs';
+import { join } from 'path';
+
 import * as CONFIG from '../../config';
 
+const getConfig = (path: string, env: string): any => {
+  const suffixPath = join(path, `${env}.config.js`);
+  const simplePath = join(path, `${env}.js`);
```

Comment:
Don't know about that either :/

Also: Update regarding the TypeScript switch. It seems this would have been switched to:

```TypeScript
  const suffixPath = join(path, `${env}.config.ts`);
  const simplePath = join(path, `${env}.ts`);
```

But this change then fails on line 16 (See note there).